# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: uriloader/exthandler/tests/mochitest/browser.ini
# Commit: 9dc679970296
# Full Hash: 9dc679970296c93353800f9765d023b24fbfc423
# Author: Sarah Ukoha <ukohasarah@gmail.com>
# Date: 2021-05-18 21:36:28
# Regressor Bug: 1555637
# File Overlap Count: 1
# Description:
#   Bug 1555637 - Ask the user what to do for local files, if the file is set to save to disk automatically r=Gijs
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D114045
# ==============================================================================

diff -r e72cd321bbaf -r 9dc679970296 uriloader/exthandler/tests/mochitest/browser.ini
--- a/uriloader/exthandler/tests/mochitest/browser.ini	Tue May 18 09:06:13 2021 +0000
+++ b/uriloader/exthandler/tests/mochitest/browser.ini	Tue May 18 09:16:28 2021 +0000
@@ -10,6 +10,9 @@
   download.bin
   download.sjs
 [browser_download_always_ask_preferred_app.js]
+[browser_local_files_no_save_without_asking.js]
+support-files =
+  file_pdf_binary_octet_stream.pdf
 [browser_download_privatebrowsing.js]
 [browser_download_open_with_internal_handler.js]
 support-files =